docs/tutorial/graphical.rst

```diff
@@ -338,5 +338,66 @@ This adds the input line above the Queue and Abort buttons.
 
 A completer is implemented allowing to quickly select an existing folder, and a button on the right side of the input widget opens a browse dialog.
 
+Using the estimator widget
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to provide estimates of the measurement procedure, an EstimatorWidget is provided that allows the user to define and calculate estimates.
+The widget is automatically activated when the :code:`get_estimates` method is added in the :code:`Procedure`.
```

Comment:
I would replace EstimatorWidget with ```:class:\`~pymeasure.display.widgets.EstimatorWidget` ``` to have a cross reference to the class documentation.
Incidentally, I have discovered that none of the classes derived from Qt stuff get document by Sphinx. Is this is a know issue ?

Reply:
Thanks, good idea.

I'll have a look at the absence of these classes in the documentation; I didn't notice this before at any rate.

Reply:
Hmmm, this is a bit weird; the docs pages of the last stable version does in fact show these classes, but the latest version doesn't. However, when I compile the docs on my own PC (both in the master branch and this branch), the classes are also there.

Reply:
I wonder whether this is linked to the changes to support PySide2.

Reply:
When I compile the docs it works in an environment with PyQT as well as in an environment with PySide2

Reply:
I think this is related to the version of Sphinx:
- v3.2.1 (Windows): problem present
- v3.5.2 (Windows): problem not present

In our CI file, we use v3.2.1, so that's why the problem is present in readthedocs latest. On the hand, it is not clear why stable version is OK.

Do you know who is looking after the CI part of the project ? Is @bilderbuchi ?

Reply:
Ahh yes it indeed is: I pinned it down to a bugfix in v3.4.2 of Sphinx:
```
#8164: autodoc: Classes that inherit mocked class are not documented
```

@bilderbuchi and @cjermain made the CI files; I'm inclined to think that we can safely update the Sphinx version; the latest version is 4.0.2, so I'd suggest to update it to that version.

Reply:
I've opened a PR to fix this issue (#460)